docs/index.asciidoc

```diff
@@ -68,6 +68,8 @@ include::management/upgrade-assistant/index.asciidoc[]
 
 include::management/dashboard_only_mode/index.asciidoc[]
 
+include::uptime/index.asciidoc[]
```

Comment:
Moved the location of the Uptime docs up a bit. Perhaps @schersh can comment on whether this is a good spot in the Kibana hierarchy?

Reply:
@bmorelli25 I suggest moving Uptime docs under APM in the docs hierarchy.  For the most part, 
the docs hierarchy follows how the apps appear in the Kibana side navigation.

Reply:
Ahh, I never noticed that. Makes sense. Thank you!

Reply:
Just out of curiosity, is there any plan to change the docs messaging to reflect that all these solutions are being nested under the Observability label?

Reply:
We're aware the current method for documenting solutions is far from ideal, and we're exploring alternatives right now! What that will look like and how long that will take remains to be seen.